nilearn/datasets/utils.py

```diff
@@ -531,7 +537,10 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
         dt = time.time() - t0
         if verbose > 0:
             # Complete the reporting hook
-            sys.stderr.write(' ...done. (%i seconds, %i min)\n' % (dt, dt // 60))
+            sys.stderr.write(' ...done. ({0:.4f} MB, {1:.0f} seconds, '
```

Comment:
I see. To be honest you already have the size in bytes on the left (in master):

```
Downloaded 314803244 of 314803244 bytes (100.0%,    0.0s remaining) ...done. (314.8032 MB, 665 seconds, 11 min)
```

so maybe just drop it.